HIP/hip-794.md

```diff
@@ -0,0 +1,65 @@
+---
+hip: 794
+title: Sunsetting Balance File
+author: Michael Heinrichs <michael@swirldslabs.com>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2023-09-12T07:00:00Z
+created: 2023-08-28
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/pull/794
+updated: 2023-08-29
+---
+
+## Abstract
+
+This HIP proposes sunsetting the balance file.
```

Comment:
Please provide more context to balance files and their purpose for informational purposes